### DISCUSSION TITLE
assertEq() Problem

### DISCUSSION THREAD

**Discussion by laradev220** [Cyfrin/foundry-full-course-cu#1143] [Q&A] [answered]
lesson 7 Testing session:

```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.18;

import {Test, console} from "forge-std/Test.sol";
import {FundMe} from "../src/FundMe.sol";

contract FundMeTest {
    FundMe fundMe;

    function setUp() external {
        fundMe = new FundMe();
    }

    function testMinimumDollarIsFive() public {
        assertEq(fundMe.MINIMUM_USD(), 5e18);
    }
}

```
The error says undeclared identifier. I don't know what to do. I go down in code checked if the assertEq are there in the Test.sol or not, founds there somewhere in the code.

Compilation error here:
```
[⠑] Compiling...
[⠢] Compiling 1 files with 0.8.19
[⠆] Solc 0.8.19 finished in 395.88ms
Error: 
Compiler run failed:
Error (7576): Undeclared identifier. Did you mean "assert"?
  --> test/FundMeTest.t.sol:15:9:
   |
15 |         assertEq(fundMe.MINIMUM_USD(), 5e18);
   |         ^^^^^^^^
```
What's the Problem with the code? The repository what i imported is unknown to me.
Thanks.

**Comment by laradev220** [comment, marked as answer]
Sorry about that. The problem is with the inheritance of program. i never inherited it. Problem solved lol

**Reply by Mahami03** [reply]
I am getting the same error. Which inheritance it is?? Can you explain it to me ??

**Reply by Mahami03** [reply]
@efffii19 Ok bro, i get it.. I have to do  'isTest' in the contract declaration. :))

**Comment by booox** [comment]
```
contract FundMeTest  is Test {
```
